### PR TITLE
selecting proposal when login, with new table style library

### PR DESCRIPTION
**Pull request by meghdadyazdi**
If login type is User, proposal table is shown only if the user is in control.
Bootsrtap library for table is now react-bootstrap-table-next

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "popper.js": "^1.16.1",
     "react": "^17.0.2",
     "react-bootstrap": "^2.3.0",
-    "react-bootstrap-table": "^4.3.1",
+    "react-bootstrap-table-next": "^4.0.3",
```
Comment (marcus-oscarsson) [inline ui/package.json]: I'm not sure react-bootstrap-table-next is compatible entirely with Bootstrap 5, it only support Bootstrap 4 at most ?

**Comment by marcus-oscarsson** [comment]
Looks good, I'm a bit hesitant towards react-bootstrap-table-next it does not seem to support Bootstrap 5 and the last version 4.3.0 was published 2 years ago ?. Do you know what the status is ?

**Comment by meghdadyazdi** [comment]
> Looks good, I'm a bit hesitant towards react-bootstrap-table-next it does not seem to support Bootstrap 5 and the last version 4.3.0 was published 2 years ago ?. Do you know what the status is ?

Seems they stoped updating it. I couldn’t find another alternative for table except the 'react-bootstrap/Table' that needs a little bit of coding and styling to get the same table in 'react-bootstrap-table-next'.

**Comment by marcus-oscarsson** [comment]
So we should probably not use  `react-bootstrap-table-next ` then if its not maintained and does not support `Bootstrap 5` which is what we are using today. Do you think you can make use of `react-boostrap/table `or some other basic component ?

**Comment by meghdadyazdi** [comment]
The final view:

proposal with title

**Comment by marcus-oscarsson** [comment]
Very nice, some spacing between the `signout` and `select proposal` would probably look better :)

Did you remove the unused dependency for the package.json ?

**Comment by meghdadyazdi** [comment]
> Very nice, some spacing between the `signout` and `select proposal` would probably look better :)
> 
> Did you remove the unused dependency for the package.json ?

I think spacing should be fixed globally (probably by Jean-Baptiste)?

I removed `react-bootstrap-table` which I think is the only dependency.

**Comment by marcus-oscarsson** [comment]
Thanks, Not certain where the "spacing" issue comes from, If its a more general problem in the bootstrap related style or something more local. You are more than welcome to have look at it even if its a general issue :). Otherwise we will merge this as it is, and we will have to take a look at a later moment.

**Comment by meghdadyazdi** [comment]
> Thanks, Not certain where the "spacing" issue comes from, If its a more general problem in the bootstrap related style or something more local. You are more than welcome to have look at it even if its a general issue :). Otherwise we will merge this as it is, and we will have to take a look at a later moment.

seems it can be solved locally

proposal with title

**Comment by marcus-oscarsson** [comment]
Excellent :)

**Pull request merged**